### ISSUE TITLE
Mail TLS is used even if value set to false

### ISSUE THREAD

**Issue by fourslashw** [mmz-srf/passbolt-helm#9]
```
root@passbolt-passbolt-helm-598bf58569-5zqql:/var/www/passbolt# printenv | grep EMAIL
EMAIL_TRANSPORT_DEFAULT_PASSWORD=
EMAIL_TRANSPORT_DEFAULT_TIMEOUT=30
EMAIL_TRANSPORT_DEFAULT_USERNAME=
EMAIL_TRANSPORT_DEFAULT_PORT=25
EMAIL_TRANSPORT_DEFAULT_HOST=ourpostfix
EMAIL_DEFAULT_FROM=passbolt@ourdomain
EMAIL_TRANSPORT_DEFAULT_TLS=false
```

```
root@passbolt-passbolt-helm-598bf58569-5zqql:/var/www/passbolt# ./bin/cake passbolt send_test_email --recipient=fourslashw@gmail.com

     ____                  __          ____  
    / __ \____  _____ ____/ /_  ____  / / /_ 
   / /_/ / __ `/ ___/ ___/ __ \/ __ \/ / __/ 
  / ____/ /_/ (__  |__  ) /_/ / /_/ / / /    
 /_/    \__,_/____/____/_.___/\____/_/\__/   

 Open source password manager for teams
---------------------------------------------------------------
 Debug email shell
---------------------------------------------------------------

Email configuration
---------------------------------------------------------------
Host: ourpostfix
Port: 25
Username: 
Password: *********
TLS: true

Sending email from: passbolt@ourdomain
Sending email to: fourslashw@gmail.com
```

Also same problem with username and password

**Comment by fourslashw** [comment]
https://github.com/mmz-srf/passbolt-helm/pull/8 possible fix

**Comment by mms-gianni** [comment]
Sorry for the late reply and thank you for your contribution. 

Looks great so far. We will merge it asap.

**Comment by mms-gianni** [comment]
It is merged now in Chart Version 1.3.1
